src/main/java/cceclipseplugin/ui/PermissionMap.java

```diff
@@ -0,0 +1,24 @@
+package cceclipseplugin.ui;
+
+import com.google.common.collect.BiMap;
+import com.google.common.collect.HashBiMap;
+
+/**
+ * Hard-coded permission map. Only to be used until the server is capable of
```

Comment:
This was left in as a fallback in case there was some issue with the request on the server, but I don't think it's necessary to have anymore.